### PR TITLE
#7 Updated main readme

### PR DESCRIPTION
**Pull request by S1mpySloth**
#7 Updated main README.md with the following:

Links to Architecture and ERD diagrams
Key decisions for Architecture/tech used/to be used
Set-up guide pointing to other READMEs
Added contributors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,81 @@
...
+EC2 provides full control over the environment for hosting a custom API. This flexibility ensures technical users can query earthquake data efficiently with tailored endpoints.
+
+7. **S3 for Long-term Data Storage:**
+S3 offers highly durable and cost-effective storage for historical earthquake data. This enables users to access historical data via Streamlit, enhancing analytical capabilities.
+
+8. **Lambda for Weekly Reports:**
+A second Lambda handles historical data archiving and report generation. It is cost-effective for periodic tasks and integrates seamlessly with SES to email reports to subscribed users.
+
+9. **SES for Email Notifications:**
+Amazon SES ensures reliable email delivery for user reports and notifications, with built-in support for monitoring delivery metrics and compliance.
```
Comment (jiuliangut) [inline README.md:52]: Could also mention SNS? we have SES for the weekly reports, then SNS for the real-time notifications

File: README.md [hunk 2]
```diff
@@ -1,5 +1,81 @@
 ![Pylint Score](.github/badges/pylint.svg)
 ![Pytest Score](.github/badges/test.svg)
 
-# earthquake-monitor
+# c14 Earthquake Monitor
 
+## Project Overview
+
+Earthquakes occur regularly, often with little warning, posing risks to life and property. Timely access to earthquake data, both real-time and historical, can empower individuals and organizations to make informed decisions, enhancing preparedness and response.
```
[the remark this answers is not in the document]
Comment (S1mpySloth) [reply README.md:8]: Absolutely, will have a scrub through to ensure no sneaky z's made it in elsewhere!

**Review by qrafiq1 — CHANGES_REQUESTED** [review]
Lovely stuff. One thing that would be nice to add in the main README is a getting started section:
- Prerequisites (Python version needed)
- Installation (git clone, setup venv, install requirements.txt command, AWS CLI.

Use this as reference: https://github.com/SurinaCS/lmnh-plant-sensors?tab=readme-ov-file#getting-started-%EF%B8%8F

**Review by qrafiq1 — APPROVED** [review]

**Review by jiuliangut — APPROVED** [review]
good job on the setup and quick-start guide - very easy to follow

**Pull request merged**